packages/iframe-execution-environment-service/src/IframeExecutionEnvironmentService.ts

```diff
@@ -176,18 +178,24 @@ export class IframeExecutionEnvironmentService
     }
 
     const job = await this._init();
+
+    let result;
+    try {
+      result = await this._command(job.id, {
+        jsonrpc: '2.0',
+        method: 'executePlugin',
+        params: pluginData,
+        id: nanoid(),
+      });
+    } catch (e) {
+      this._deleteJob(job.id);
```

Comment:
calling `this._init()` above this line adds the job, so when an error happens on execution we should remove it.